### DISCUSSION TITLE
infinite loop/ memory leak /  unintentionally reloading the DOM

### DISCUSSION THREAD

**Discussion by JuanDavila1101** [nss-evening-cohort-14/evening-client-side#305] [Capstones!] [answered]
I see that Mathew  experienced a similar issue with on a prior ticket 

https://github.com/nss-evening-cohort-14/evening-client-side/discussions/281
index.js:1 Warning: unstable_flushDiscreteUpdates: Cannot flush updates when React is already rendering.

the error keeps occurring until the stack runs out of resources. 

the Stack overflow explains this 
https://stackoverflow.com/questions/58040892/cannot-flush-updates-when-react-is-already-rendering
https://github.com/facebook/react/issues/20141

I merged a branch with the error here
https://github.com/JuanDavila1101/workersonsite/blob/main/src/components/SitesForm.js
git@github.com:JuanDavila1101/workersonsite.git

I think line 49 keeps getting hit, but I don't know why

**Comment by TrinityChristiana** [comment]
We need to at least know what view/file is causing the error. Can you please update your links with a specific view/file?

**Reply by JuanDavila1101** [reply]
updated the link to this https://github.com/JuanDavila1101/workersonsite/blob/main/src/components/SitesForm.js

**Comment by TrinityChristiana** [comment, marked as answer]
@JuanDavila1101 Will add their answer!

**Reply by JuanDavila1101** [reply]
The error was probably fixed during testing an anonymous function but having too many debuggers was hiding the actual code fix, had to hide all debuggers and hard refresh and all errors went away and the anonymous function acted as intended.

Thank you Trinity!!!
